client/src/components/AlumniDirectory.js

```diff
@@ -13,22 +13,22 @@ const searchOptions = [
     {
         key: 'Location',
         text: 'Location',
-        value: 'location'
+        value: 'country'
     },
     {
         key: 'College',
         text: 'College',
-        value: 'college'
+        value: 'collegeName'
```

Comment:
Since we are storing the mongoIds for indexable fields on the alumni records, the value of these fields are now stored as `*Name` fields on the records